iamproperty/iamproperty.github.io
# Logo brand updates

## Summary of Changes:
1. Update CSS to in-force brand rules
2. Update documentation

## Rationale
Update existing logo styles to match the new brand guidelines

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /foundations/logos

## Ticket(s)
DS-125

## Checklist

## Notes
